### ISSUE TITLE
DisallowPartiallyKeyed: false positive with array unpacking

### ISSUE THREAD

**Issue by janedbal** [slevomat/coding-standard#1552]
Following is reported as violation of `SlevomatCodingStandard.Arrays.DisallowPartiallyKeyed`:
```php
$config = ['foo' => 'bar'];
$newConfig = ['key' => 'value', ... $config]; // reported
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/91428d5bcf7db93a842bcf97f465edf62527f3ea

**Comment by janedbal** [comment]
Thanks!

**Comment by bkdotcom** [comment]
@kukulich   Thanks!
